tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -82,6 +84,21 @@ var _ = Describe("odo devfile create command tests", func() {
 		})
 	})
 
+	Context("When executing odo create with devfile component type argument and --context flag", func() {
+		It("should successfully create the devfile componet in the context", func() {
+			newContext := path.Join(context, "newContext")
+			devfilePath := filepath.Join(newContext, "devfile.yaml")
+			envFilePath := filepath.Join(newContext, ".odo/env/env.yaml")
+			helper.MakeDir(newContext)
```

Comment:
@GeekArthur +1, however you need to add the cleanup steps for the dir ```newContext``` at the end of this spec. For example you just need to add ```helper.DeleteDir(newContext)``` after line number 98.

Reply:
@amitkrout Actually `newContext` is subdir of `context` because of its assignment here `newContext := path.Join(context, "newContext")`, and we clean up the `context` dir directly here https://github.com/openshift/odo/pull/2917/files#diff-5291a6a42f2148d4c3e48ea4b73cff23L33 in `AfterEach` statement, which means `newContext` dir cleanup will be covered by `context` dir cleanup. That being said, I can still add `newContext` cleanup explicitly for readability.